portal-web/test/functional/com/liferay/portalweb/tests/myfirsttest.testcase

```diff
@@ -0,0 +1,20 @@
+<definition component-name=<"portal-smoke">
+	<property name="testray.main.component.name" value="Smoke" />
+	
+	<command name="SuperAdminLogin" priority="5">
+		<execute function="Open" locator1="http://localhost:8080" />
+		<execute function="waitForPageToLoad" value="3000" />
+		<execute function="click" locator1="//button[contains(.,'Sign In')]"
+		<execute function="type" locator1="//input[@id="_com_liferay_login_web_portlet_LoginPortlet_login"]" value1="test@liferay.com" />
+		<execute function="type" locator1="//input[@id="_com_liferay_login_web_portlet_LoginPortlet_password"]" value1="test" />
+		<execute function="click" locator1="//button[contains(., 'Sign In')]"
```

Comment:
(., 'Sign In') technically works, but to keep things stylistically uniform, we want to remove the extra space so that it is just like line 7 (.,'Sign In')